server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -37,9 +39,14 @@
 import us.freeandfair.corla.model.*;
 import us.freeandfair.corla.model.CastVoteRecord.RecordType;
 import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.CastVoteRecordQueries;
 import us.freeandfair.corla.query.CountyContestResultQueries;
 import us.freeandfair.corla.util.DBExceptionUtil;
 import us.freeandfair.corla.util.ExponentialBackoffHelper;
+import us.freeandfair.corla.util.IRVParsingException;
+
+import static us.freeandfair.corla.model.ContestType.IRV;
```

Comment:
Rather than importing the specific enumeration case ContestType.IRV, just import ContestType and then use ContestType.IRV where necessary.